sdcm/cluster_k8s/gke.py

```diff
@@ -191,11 +198,11 @@ def __init__(self,
         self.gce_disk_type = gce_disk_type
         self.gce_disk_size = gce_disk_size
         self.gce_network = gce_network
-        self.gce_services = services
+        self.gce_service, info = gce_service
```

Comment:
If the `info` is not used then shouldn't it be `_`? How does the `Pylint` not complain about it?

Reply:
it is used few line below